apps/web/src/app/(private)/datasets/page.tsx

```diff
@@ -1,3 +1,100 @@
-export default function DatasetsRoot() {
-  return null // --> layout.tsx
+import {
+  DatasetsRepository,
+  DatasetsV2Repository,
+} from '@latitude-data/core/repositories'
+import { TableWithHeader } from '@latitude-data/web-ui'
+import { getCurrentUser } from '$/services/auth/getCurrentUser'
+import env from '$/env'
```

Comment:
don't use this file it will get removed use @latitude-data/env please

Reply:
Yep, changed 
![image](https://github.com/user-attachments/assets/7d7a54b7-a5fa-48b4-abef-6e7d5bca5685)